linux_os/guide/services/ntp/chronyd_or_ntpd_set_maxpoll/rule.yml

```diff
@@ -11,6 +11,8 @@ description: |-
     <tt>maxpoll</tt> in <tt>/etc/ntp.conf</tt> or <tt>/etc/chrony.conf</tt>
     add the following:
     <pre>maxpoll {{{ xccdf_value("var_time_service_set_maxpoll") }}}</pre>
+    to <pre>server</pre> direstives. If using chrony any <pre>pool</pre> directives
```

Comment:
typo: direstives

Reply:
Addressed in https://github.com/ComplianceAsCode/content/pull/7945